README.md

```diff
@@ -4,76 +4,122 @@
+The Dependency Analytics extension uses the [Snyk Vulnerability Database](https://snyk.io/product/vulnerability-database/) for the most up-to-date vulnerability information available.
+Snyk uses industry-leading security intelligence by pulling from many data sources to give you exact vulnerability information, along with advice on how to remediate those security risks.
+
+**NOTE:**
+<br >The Red Hat Dependency Analytics extension is an online service hosted and maintained by Red Hat.
+Dependency Analytics only accesses your manifest files to analyze your application dependencies before displaying the vulnerability report.
+
+**IMPORTANT:**
+<br >Currently, Dependency Analytics only supports projects that use Maven (`mvn`), and Node ecosystems (`npm`).
+In future releases, Red Hat plans to support other programming languages.
+
+##### Table of Contents
+- [Quick start](#quick-start)
+- [Configuration](#configuration)
+- [Features](#features)
+- [Using Red Hat Dependency Analytics for CI builds](#using-red-hat-dependency-analytics-for-ci-builds)
+- [Know more about the Red Hat Dependency Analytics platform](#know-more-about-the-red-hat-dependency-analytics-platform)
+- [Data and telemetry](#data-and-telemetry)
+- [Support, feedback \& questions](#support-feedback--questions)
+- [License](#license)
+
+## Quick start
+
+**Prerequisites**
+
+- For Maven projects, analyzing a `pom.xml` file, you must have the `mvn` binary in your system’s `PATH` environment.
+- For Node projects, analyzing a `package.json` file, you must have the `npm` binary in your system’s `PATH` environment.
+
+<br >**IMPORTANT:** 
+<br >Visual Studio Code by default executes binaries directly in a terminal found in your system's `PATH` environment.
+You can configure Visual Studio Code to look somewhere else to run the necessary binaries.
+You can configure this by accessing the [extension settings](https://code.visualstudio.com/docs/getstarted/settings).
+Click the **Workspace** tab, search for the word _executable_, and specify the absolute path to the binary file you want to use for Maven or Node.
+
+**Procedure**
+
+1. Install [Visual Studio Code](https://code.visualstudio.com/) on your workstation.
+2. After the installation finishes, open the Visual Studio Code application.
+3. Click the **Extensions** icon from the **Activity Bar** or click **View** from the file menu, and click **Extensions**.
+4. Search the **Marketplace** for _Dependency Analytics_.
+5. Click the **Install** button to install the extension.
+6. To start scanning your application for security vulnerabilities, and view the vulnerability report, you can do one of the following:
+   - Open a manifest file, hoover over a dependency marked by the inline analysis, indicated by the wavy-red line under a version number or dependency name, click **Quick Fix**, and click **Detailed Vulnerability Report**.
+   - Open a manifest file, and click the **pie chart icon** ![ Pie chart icon ](images/0.2.0/icon.png).
```

Comment:
I am not sure if the link to the icon is valid

Reply:
yes, true, I have recently changed the images directory. 
You can leave it as is and ill fix the links after the PR merge